src/test/java/io/seqera/tower/cli/BaseCmdTest.java

```diff
@@ -47,8 +47,16 @@
     Path tempDir;
 
     protected byte[] loadResource(String name) {
+        return loadResource(name, "json");
+    }
+
+    protected byte[] loadResource(String name, String ext) {
         try {
-            return this.getClass().getResourceAsStream("/runcmd/" + name + ".json").readAllBytes();
+            InputStream stream = this.getClass().getResourceAsStream("/runcmd/" + name + "." + ext);
```

Comment:
Since Java 7 this can be done like this:
```java 
try (InputStream stream = this.getClass().getResourceAsStream("/runcmd/" + name + "." + ext)) {
    return stream.readAllBytes();
}
```
And it is automatically close.